lib/linter/linter.js

```diff
@@ -2016,7 +2016,7 @@ class Linter {
     }
 
     getFixPasses() {
-        return internalSlotsMap.get(this).fixPasses;
+        return internalSlotsMap.get(this).fixPasses || 0;
```

Comment:
Would it be more appropriate to use `??` here? (I'm assuming `fixPasses` is either missing or a number.)